### PR TITLE
Options for multiple paths

### PR DESCRIPTION
**Pull request by mrksr**
This implements issue #995. I have not yet tested the PR with python 2.
I'm very interested in your thoughts concerning coding style, since I have tried to follow your lead.

### CODE REVIEW

File: holoviews/ipython/parser.py [hunk 1]
```diff
@@ -268,30 +313,39 @@ def parse(cls, line, ns={}):
...
-                parse[group['pathspec']].update(options)
-            else:
-                parse[group['pathspec']] = options
-        return parse
+                options['style'] = {cls.aliases.get(k,k):v for k,v in opts.items()}
+
+            for pathspec in pathspecs:
+                if pathspec not in parse:
+                    parse[pathspec] = {}
+                cls._update_options(parse[pathspec], options)
```
Comment (jlstevens) [inline holoviews/ipython/parser.py]: Perhaps something like ``parse[pathspec] = merged_options(parse.get(pathspec, {}), options)`` would be nicer?

**Comment by jlstevens** [comment]
Looks great, thanks for the contribution and thanks for all the new unit tests!

You have already addressed most of my comments and I am ready to merge once you address my latest suggestion (renaming ``_update_options``). The tests are passing so I think it looks good to go!

**Comment by mrksr** [comment]
Will do. Thank you for your guidance :-)

**Comment by mrksr** [comment]
Rename should be done now and I added one more test to reflect our decision on precedence in merges.

I had to replace the commits with new author information which includes my private mail, so do not be confused that everything is new.

**Comment by jlstevens** [comment]
All looks good so I am now merging. Thanks once again for contributing!

**Pull request merged**